imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -12,57 +12,143 @@ import { setDrawerState } from '../general-actions'
+              <MenuItem key={id} innerDivStyle={{padding: 0}} onClick={() => dispatch(push(`/unit/${id}`))}>
+                <div className='mt2 ph2 bb b--very-light-gray br1 w-100 flex items-center pa2'>
+                  <FontIcon className='material-icons' color='var(--semi-dark-gray)'>home</FontIcon>
+                  <div className='ml3 mv1 semi-dark-gray lh-copy flex-grow overflow-hidden'>
+                    <div className='ti1 ellipsis'>{metaData.displayName || name}</div>
+                    <div className='ti1 ellipsis silver'>{ metaData.moreInfo || description}&nbsp;</div>
+                  </div>
+                  <div
+                    onClick={evt => {
+                      evt.stopPropagation()
+                      dispatch(push(`/case/new?unit=${id}`))
+                    }}
+                    className='f6 ellipsis ml3 pl1 mv1 bondi-blue fw5 no-shrink'
+                  >
+                    Add case
+                  </div>
+                </div>
+              </MenuItem>
+            ))}
+        </div>
+        <div>
+          <div className='bb b--black-10 bg-very-light-gray f6 fw5 mid-gray pv2 pl2'>
+             Units Involved In
+          </div>
+          {
+            unitsInvolvedIn.map(({ id, name, description, metaData }) => (
+              <MenuItem key={id} innerDivStyle={{padding: 0}} onClick={() => dispatch(push(`/unit/${id}`))}>
+                <div className='mt2 ph2 bb b--very-light-gray br1 w-100 flex items-center pa2'>
+                  <FontIcon className='material-icons' color='var(--semi-dark-gray)'>home</FontIcon>
+                  <div className='ml3 mv1 semi-dark-gray lh-copy flex-grow overflow-hidden'>
+                    <div className='ti1 ellipsis'>{metaData.displayName || name}</div>
+                    <div className='ti1 ellipsis silver'>{ metaData.moreInfo || description}&nbsp;</div>
+                  </div>
+                  <div
+                    onClick={evt => {
+                      evt.stopPropagation()
+                      dispatch(push(`/case/new?unit=${id}`))
+                    }}
+                    className='f6 ellipsis ml3 pl1 mv1 bondi-blue fw5 no-shrink'
+                  >
+                  Add case
+                  </div>
+                </div>
+              </MenuItem>
+            ))}
+        </div>
+      </div>
+      }
+    </div>
+  )
+}
 
 class UnitExplorer extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      slideIndex: 0,
+      activeStatus: true
+    }
+  }
+
+  handleChange = (value) => {
+    this.setState({
+      slideIndex: value,
+      activeStatus: !this.state.activeStatus
+    })
+  };
+
   render () {
     const { isLoading, unitList, dispatch } = this.props
+    const activeDisabledFilter = this.state.activeStatus ? unitItem => !unitItem.metaData.disabled : unitItem => unitItem.metaData.disabled
```

Comment:
You can do away with this if you just have two lists, each sent to its component

Reply:
removed